src/main/java/trackitnus/model/TrackIter.java

```diff
@@ -249,10 +251,6 @@ public void removeLesson(Lesson key) {
         lessons.remove(key);
     }
 
-    public void sortLesson() {
-        lessons.sort();
-    }
-
```

Comment:
Why is this removed? Is it because not called at all? Minh wanted to keep this in initially so maybe should discuss w him before removing it

Reply:
It's moved haha